src/modules/simaris.js

```diff
@@ -13,8 +13,8 @@ var Simaris = function(data) {
     {
       return;
     }
-    if(!data.LastCompletedTargetType){
-      this.target = strings[data.LastCompletedTargetType].value;  
+    if(data.LastCompletedTargetType){
+      this.target = dsUtil.getLocalized(data.LastCompletedTargetType.toLowerCase());

```

Comment:
should i move the `.toLowerCase()` to the util call, since it's going to be needed to call it every time?